src/openrct2/world/Sprite.h

```diff
@@ -32,11 +31,12 @@ enum SPRITE_IDENTIFIER
 enum SPRITE_LIST
 {
     SPRITE_LIST_FREE,
-    SPRITE_LIST_TRAIN,
+    SPRITE_LIST_VEHICLE_HEAD,
```

Comment:
Is vehicle head clear as to what it means? Train Head might be more clear. I'm not saying it should be changed just wanting to discuss it.

Reply:
I picked vehicle because the it covers quite a few things, the game does not only have trains as in railed vehicles. But I can change it to train if that makes more sense.